### PR TITLE
Picture

### PR DESCRIPTION
**Pull request by Sonya092**

### CODE REVIEW

File: src/components/Glaucoma/index.jsx [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import DropdownWindow from 'components/DropdownWindows';
+import { useLocation } from 'react-router-dom';
 
-export default function Glaucoma({ state }) {
+export default function Glaucoma() {
+  const [glaucomas, setGlaucomas] = useState([]);
+  useEffect(() => {
+    axios.get(`http://localhost:5050/contents`).then(({ data }) => {
```
Comment (KeerhemFR) [inline src/components/Glaucoma/index.jsx:9]: Will have to be replaced by the key calling localhost in dotenv after the merge

File: src/components/Cataract/index.jsx [hunk 2]
```diff
@@ -1,35 +1,39 @@
...
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import DropdownWindow from 'components/DropdownWindows';
 
-function Cataract({ state }) {
+export default function Cataract() {
+  const [cataracts, setCataracts] = useState([]);
+  useEffect(() => {
+    axios.get(`http://localhost:5050/contents`).then(({ data }) => {
```
Comment (KeerhemFR) [inline src/components/Cataract/index.jsx:9]: Will have to be replaced by the key calling localhost in dotenv after the merge

File: src/components/Picture/index.jsx [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import axios from 'axios';
 import { useEffect, useState } from 'react';
+import { PropTypes } from 'prop-types';
 import SPicture from './style';
 
-export default function Pictures() {
+export default function Pictures({ idContent }) {
   const [pictures, setPictures] = useState([]);
   useEffect(() => {
     axios.get(`http://localhost:5050/images`).then(({ data }) => {
```
Comment (KeerhemFR) [inline src/components/Picture/index.jsx:9]: Will have to be replaced by the key calling localhost in dotenv after the merge

File: src/components/Refractive/index.jsx [hunk 4]
```diff
@@ -1,34 +1,39 @@
...
+import { useEffect, useState } from 'react';
 import DropdownWindow from 'components/DropdownWindows';
-import refractiveContent from './content';
+import { useLocation } from 'react-router-dom';
 
-export default function Refractive({ state }) {
+export default function Refractive() {
+  const [refractives, setRefractives] = useState([]);
+  useEffect(() => {
+    axios.get(`http://localhost:5050/contents`).then(({ data }) => {
```
Comment (KeerhemFR) [inline src/components/Refractive/index.jsx:9]: Will have to be replaced by the key calling localhost in dotenv after the merge

**Review by KeerhemFR — APPROVED** [review]
See no evil, but will have to correct the path after being merge so the path is not visible in the code.

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**